src/System.Security.Cryptography.X509Certificates/src/System/Security/Cryptography/X509Certificates/Asn1/TbsCertificateAsn.xml

```diff
@@ -0,0 +1,38 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<asn:Sequence
+  xmlns:asn="http://schemas.dot.net/asnxml/201808/"
+  name="TbsCertificateAsn"
+  namespace="System.Security.Cryptography.Asn1">
```

Comment:
namespace => System.Security.Cryptography.X509Certificates.Asn1

Reply:
This was partly intentional, but didn't necessary have to be changed, so I guess I will go back and restore the namespaces. I intend to eventually move `CertificateAsn`, `TbsCertificateAsn`, `ValidityAsn` and `TimeAsn` to common code and remove the duplicate (and incomplete) definitions in test code.

There was some precedent in the S.S.C.Algorithms, which already used this namespace instead of its own.

Reply:
...or, on second thought, I don't have to move the definitions and I can move the test code (X509DeserializeTests.cs) and get rid of duplicate test data at the same time. Not sure, what is preferable.